src/EnergyPlus/Furnaces.cc

```diff
@@ -9373,107 +9297,6 @@ namespace Furnaces {
-        // operation. ASHRAE Transactions 102 (1): 266-274
-
-        // (2) Henderson, H.I. Jr., Y.J. Huang and Danny Parker. 1999. Residential Equipment
-        // Part Load Curves for Use in DOE-2.  Environmental Energy Technologies Division,
-        // Ernest Orlando Lawrence Berkeley National Laboratory.
-
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 PartLoadFactor; // Part load factor
-        Real64 PLF2;           // (i+1)th term of part load factor
-        Real64 A;              // Variable for simplify equation
-
-        Real64 Nmax = state.dataFurnaces->Furnace(FurnaceNum).MaxONOFFCyclesperHour;
-        Real64 tau = state.dataFurnaces->Furnace(FurnaceNum).HPTimeConstant;
-        Real64 pr = state.dataFurnaces->Furnace(FurnaceNum).OnCyclePowerFraction;
-
-        // Initialize
-        errFlag = false;
-        Real64 error = 1.0;
-        int NumIteration = 0;
-
-        // Initial guess for part load fraction
-        Real64 PLF1 = 1.0;
-
-        // Calculate PLF using successive substitution until convergence is achieved
-        while (true) {
-            ++NumIteration;
-
-            if (PLR == 1) {
-                // Set part load fraction, PLF1=1.0 if PLR=1.0 and exit loop
-                PLF1 = 1.0;
-                goto LOOPPLF_exit;
-            }
-
-            if (NumIteration > 100) {
-                // Exit loop if interation exceed 100
-                errFlag = true;
-                PLF1 = 1.0;
-                goto LOOPPLF_exit;
-            }
-
-            if (error < 0.00001) {
-                // Exit loop if convergence is achieved
-                goto LOOPPLF_exit;
-
-            } else {
-                // Calculate PLF
-                A = 4.0 * tau * (Nmax / 3600.0) * (1 - PLR / PLF1);
-                if (A < 1.5e-3) {
-                    // A safety check to prevent PLF2 = 1 - A * (1 - Exp(-1 / A))
-                    // from "float underflow error". Occurs when PLR is very close to 1.0,
-                    // small A value, thus Exp(-1/A) = 0
-                    PLF2 = 1 - A;
-                } else {
-                    PLF2 = 1.0 - A * (1.0 - std::exp(-1.0 / A));
-                }
-                error = std::abs((PLF2 - PLF1) / PLF1);
-                PLF1 = PLF2;
-            }
-        }
-    LOOPPLF_exit:;
-
-        // Adjust PLF for the off cycle power consumption if
-        // on-cycle power use is specified by the user
-        if (pr > 0.0) {
-            PartLoadFactor = PLR / ((PLR / PLF1) + (1 - PLR / PLF1) * pr);
-        } else {
-            PartLoadFactor = PLF1;
-        }
-
-        if (PartLoadFactor <= 0.0) {
-            PartLoadFactor = 0.0;
-            RuntimeFrac = 0.0;
-            errFlag = true;
-        } else {
-            RuntimeFrac = PLR / PartLoadFactor;
-        }
-
-        if (RuntimeFrac > 1.0) {
-            RuntimeFrac = 1.0;
-        }
-    }
```

Comment:
This entire function is now eliminated because PLF and RTF are calculated by the child coils based on the coil PLF curve.